ClinGen/clincoded
# Connects to #322. Connects to #475. Connects to #613. Connects to #622. Connects to #629. Text and Assessment display changes

#### Connects to #322:

Changes to text in Family Submit page:

![image](https://cloud.githubusercontent.com/assets/4326866/13127822/6f953fa0-d586-11e5-85c9-47508cffc6ab.png)

![image](https://cloud.githubusercontent.com/assets/4326866/13158130/5c1d5114-d640-11e5-9b98-ecfe1650384b.png)

### Connects to #475:

Changes to how the Assessments and Assessment Panels are displayed on Family, Experimental Data, and Variant View pages.
Also adds notice when owner of object has not assessed.

#### Experimental Data

Non-owner, looking at not-assessed object:
![image](https://cloud.githubusercontent.com/assets/4326866/13192445/d70f2e84-d720-11e5-9ae2-8b76329a750c.png)

Non-owner, looking at assessed object:
![image](https://cloud.githubusercontent.com/assets/4326866/13192454/edcd1ab4-d720-11e5-9e4c-01cfef1af4da.png)


#### Family Segregation Data:

Non-owner, looking at not-assessed object:
![image](https://cloud.githubusercontent.com/assets/4326866/13192463/11152bc4-d721-11e5-9cce-b55e7f8428f0.png)

Non-owner, looking at assessed object:
![image](https://cloud.githubusercontent.com/assets/4326866/13192470/22ffe19e-d721-11e5-9479-83bdd8c1cfe6.png)


#### Variant Data:

Curate Variant Assessment Panel (no change...):
![image](https://cloud.githubusercontent.com/assets/4326866/13192480/4917b546-d721-11e5-9130-799b3a12e4b6.png)

Assessments showing up above and below (other user's assessment):
![image](https://cloud.githubusercontent.com/assets/4326866/13192495/7ef34522-d721-11e5-9446-44f7bcfe54f4.png)

Assessments showing up above and below (other user's and own assessment):
![image](https://cloud.githubusercontent.com/assets/4326866/13192500/9a16b17c-d721-11e5-9204-e2802d9cc5a6.png)

### Connects to #613:

Changes to layout and text in Group submit page:

![image](https://cloud.githubusercontent.com/assets/4326866/13160687/d3d24e32-d64d-11e5-86fb-53ab911aa4d6.png)
![image](https://cloud.githubusercontent.com/assets/4326866/13160694/dbe86cfa-d64d-11e5-874d-d9469c21bf32.png)



### Testing:

Only changes regarding #475 require testing, as the others are text/css changes.

1. Create GDM, Add PMID
2. Create Experimental Data, Family w/ Segregation, and Variant objects
3. View these objects as another user to confirm that the displays are correct (Assessment Panel exists but is disabled, and a blue notice box explaining why is visible for Experimental Data and Family)
4. Assess on these objects w/ original user
5. Confirm correct display of these items' View pages with another user.
